backend/src/run.py

```diff
@@ -232,9 +232,8 @@ async def run_individual(request: Request):
             )
         # Cache the output of the node
         request.app.ctx.cache[full_data["id"]] = output
-        extra_data = node_instance.get_extra_data()
         del node_instance, run_func
-        return json({"success": True, "data": extra_data})
+        return json({"success": True, "data": {}})
```

Comment:
I think this will make a cleaner API. Also, please change the signature of `Backend#runIndividual` to `runIndividual(data: BackendRunIndividualRequest): Promise<BackendResult<null>>`.

```suggestion
        return json({"success": True, "data": None})
```